domain-service/credit-application/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.2</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <groupId>at.deloittedigital</groupId>
+    <artifactId>credit-application</artifactId>
```

Comment:
Why not `credit-service`?

Reply:
Well, its full name should be `credit-application-service` then :) Note that `application` here refers to applicants for the credit a.k.a. clients of the bank. But I could add `service` suffix, sure.

Reply:
Ah, yes please, it might be confusing otherwise.

Reply:
👍